packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -146,33 +197,73 @@ class Accordion extends AutoControlledComponent<WithAsProp<AccordionProps>, any>
       _.invoke(predefinedProps, 'onClick', e, titleProps)
       _.invoke(this.props, 'onTitleClick', e, titleProps)
     },
+    onFocus: (e: React.SyntheticEvent, titleProps: AccordionTitleProps) => {
+      _.invoke(predefinedProps, 'onFocus', e, titleProps)
+      this.setState({ focusedIndex: titleProps.index })
+    },
   })
 
-  isIndexActive = (index): boolean => {
+  private isIndexActive = (index: number): boolean => {
     const { exclusive } = this.props
     const { activeIndex } = this.state
 
     return exclusive ? activeIndex === index : _.includes(activeIndex, index)
   }
 
+  private isIndexClosingPrevented = (index: number): boolean => {
```

Comment:
would be good to add comments about what this method does, not very self-explanatory method name

Reply:
I had a hard time to think about a proper name for it. I will probably name it `canBeCollapsed` as per Marija's suggestion below :D. Will also add description thanks!

Reply:
I like `canBeCollapsed` more 😄